pkgs/top-level/all-packages.nix

```diff
@@ -42283,4 +42283,6 @@ with pkgs;
   ssl-proxy = callPackage ../tools/networking/ssl-proxy { };
 
   code-maat = callPackage ../development/tools/code-maat {};
+
```

Comment:
this should probably use python3's callPackage to build properly. did you test this?

Reply:
Oh yeah, I have this locally, forgot to add it to the PR.